woocommerce/woocommerce-ios
# [Mobile Payments] Remove Hardware and dependency with the StripeTerminal SDK

Closes #3681 

For more context: p1613553100463900-slack-mobile-woo

We need to rethink the way we implement the integration with the Stripe Terminal SDK in a way that it is safe for production builds. We do not want to risk prompting users for Bluetooth or location permissions at this point.

After several discussions and several attempts at finding the optimal way to do so, we have decided to remove the integration from develop, and continue development in a feature branch. (see, for example, #3656 and #3682)

## Changes
* Remove Hardware package and its contents.
* Remove dependency with Stripe Terminal SDK.
* Remove CardPresentPaymentStore and CardPresentPaymentAction.
* Remove instantiation of CardPresentPaymentsStore from AuthenticatedState.
* Remove HardwareTests from WooCommerce’s testing plan.

## How to test
* Check out the branch, run `bundle exec pod install`
* Build the project. Make sure the project builds without issues on the simulator
* Run the tests. Make sure all the tests pass.
* Build the project on a device. There shouldn't be any errors

Update release notes:
